src/EnergyPlus/WindowManager.hh

```diff
@@ -74,312 +74,230 @@ namespace WindowManager {
-                             int const SurfNum,          // Surface number
+                             int SurfNum,                // Surface number
                              Array1A<Real64> AbsRadShade // Short-wave radiation absorbed by shade/blind faces
     );
 
-    //****************************************************************************
-
     void NusseltNumber(EnergyPlusData &state,
-                       int const SurfNum, // Surface number
-                       Real64 const tso,  // Temperature of gap surface closest to outside (K)
-                       Real64 const tsi,  // Temperature of gap surface closest to zone (K)
-                       int const IGap,    // Gap number
-                       Real64 const gr,   // Gap gas Grashof number
-                       Real64 const pr,   // Gap gas Prandtl number
-                       Real64 &gnu        // Gap gas Nusselt number
+                       int SurfNum, // Surface number
+                       Real64 tso,  // Temperature of gap surface closest to outside (K)
+                       Real64 tsi,  // Temperature of gap surface closest to zone (K)
+                       int IGap,    // Gap number
+                       Real64 gr,   // Gap gas Grashof number
+                       Real64 pr,   // Gap gas Prandtl number
+                       Real64 &gnu  // Gap gas Nusselt number
     );
 
-    //*******************************************************************************************************
-
-    void TransAndReflAtPhi(Real64 const cs,                // Cosine of incidence angle
-                           Real64 const tf0,               // Transmittance at zero incidence angle
-                           Real64 const rf0,               // Front reflectance at zero incidence angle
-                           Real64 const rb0,               // Back reflectance at zero incidence angle
-                           Real64 &tfp,                    // Transmittance at cs
-                           Real64 &rfp,                    // Front reflectance at cs
-                           Real64 &rbp,                    // Back reflectance at cs
-                           bool const SimpleGlazingSystem, // .TRUE. if simple block model being used
-                           Real64 const SimpleGlazingSHGC, // SHGC value to use in alternate model for simple glazing system
-                           Real64 const SimpleGlazingU     // U-factor value to use in alternate model for simple glazing system
+    void TransAndReflAtPhi(Real64 cs,                // Cosine of incidence angle
+                           Real64 tf0,               // Transmittance at zero incidence angle
+                           Real64 rf0,               // Front reflectance at zero incidence angle
+                           Real64 rb0,               // Back reflectance at zero incidence angle
+                           Real64 &tfp,              // Transmittance at cs
```

Comment:
Consider using a `struct` with `tf`, `rf`, and `rb` to reduce parameter passing.

Reply:
💯